iscript/src/iscript/mac.py

```diff
@@ -257,7 +257,10 @@ async def sign_app(sign_config, app_path, entitlements_path, provisioning_profil
 
         for file_ in files:
             abs_file = os.path.join(top_dir, file_)
-            await _do_sign_file(top_dir, abs_file, file_, sign_command, app_path_len, app_executable)
+            if sign_config.get("sign_with_entitlements", False) or file_ in sign_config.get("no_entitlements_files", []):
```

Comment:
shouldn't this be `if not sign_config.get("sign_with_entitlements", False) or ...` ?

Reply:
ummm, oops!